.github/workflows/ci.yml

```diff
@@ -308,13 +308,79 @@ jobs:
           --qserv-image ${{ needs.image-names.outputs.qserv-image }} \
           --mariadb-image ${{ needs.image-names.outputs.mariadb-image }}
 
+      - name: Check Qserv containers
+        run: sleep 180 && docker ps -a
+
       - name: Run integration tests
         run: |
           ./admin/local/cli/qserv --log-level DEBUG itest \
-          --wait 180 \
           --qserv-image ${{ needs.image-names.outputs.qserv-image }} \
           --mariadb-image ${{ needs.image-names.outputs.mariadb-image }}
 
+      - name: Check Qserv containers
+        if: always()
```

Comment:
Are the `if: always()` needed here?

Reply:
I'm not 100% sure. My understanding was that the precondition allows running a step regardless of the completion status of the previous step. Perhaps it's too aggressive. I hope it won't hurt anything(?)